app/widget/containers/main_body/content/settings/settings_tabs/team_member_tab/team_member_table_group.py

```diff
@@ -0,0 +1,104 @@
+from PySide2.QtCore import QAbstractTableModel, Qt
+from PySide2.QtWidgets import (
+    QAbstractItemView,
+    QGroupBox,
+    QHeaderView,
+    QTableView,
+    QVBoxLayout,
+    QWidget,
+)
+from schemas.db_schema import TeamMemberDbSchema
+
+from app.db.queries import get_team_members, upsert_team_members
+
+
+class TeamMemberTable(QAbstractTableModel):
+    def __init__(self):
+        super().__init__()
+        self.horizontal_headers = ["id", "imię i nazwisko", "specjalność"]
+        self._data = [[tm.id, tm.name, tm.function] for tm in get_team_members()]
+
+    def data(self, index, role):
+        if role == Qt.DisplayRole:
+            return self._data[index.row()][index.column()]
+
+    def setData(self, index, value, *args):
+        self._data[index.row()][index.column()] = value
+        self.dataChanged.emit(index, index, [Qt.DisplayRole])
+        return True
+
+    def is_last_row_valid(self) -> bool:
+        last_team_member = self._data[-1]
+        return last_team_member[1] and last_team_member[2]
+
+    def append_new_team_member(self):
+        last_team_member = self._data[-1]
+        if self.is_last_row_valid():
+            new_team_member = [last_team_member[0] + 1, "", ""]
+            self._data.append(new_team_member)
+            self.layoutChanged.emit()
+
+    def remove_team_member(self, row: int):
+        self._data.pop(row)
+        self.layoutChanged.emit()
+
+    def rowCount(self, index):
```

Comment:
`camelCase`? ;)

Reply:
This metod it's part of this model API: https://doc.qt.io/qtforpython-6/PySide6/QtCore/QAbstractTableModel.html#detailed-description

Need to be named like this.